mne/stats/regression.py

```diff
@@ -246,6 +248,16 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
     """
 
     if isinstance(solver, string_types):
+        if solver == 'cholesky':
+            try:
+                from sklearn.linear_model import Ridge
+                def solver(X, y):
+                    return Ridge(fit_intercept=False, copy_X=False, alpha=0.,
+                                 solver='cholesky').fit(X, y.T).coef_
```

Comment:
The parameter is already there actually - it's important, because it allows to plug in default solvers. But in principle, 'pinv' could be replaced with 'cholesky' - in my personal tests, it's always much faster - but I haven't done a systematic check.